tests/conftest.py

```diff
@@ -211,13 +210,13 @@ def create_dependency_fixture(
         dependency_bin.chmod(0o755)
         return dependency_bin
 
-    yield create_dependency_fixture
+    return create_dependency_fixture
 
 
 ChmodCall = collections.namedtuple("ChmodCall", ["owner", "group", "kwargs"])
 
 
-@pytest.fixture
+@pytest.fixture()
```

Comment:
Any special reason not to use `dict` in the line below?

Reply:
Just keep the old style for now